xml/admin_gui_dashboard.xml

```diff
@@ -798,32 +905,37 @@
     </mediaobject>
    </figure>
    <para>
-    Click a node name in the <guimenu>Host</guimenu> column to view an extended
-    table with details about the OSD setting and performance. Browsing through
+    Click the drop-down arrow next to a node name in the <guimenu>Host</guimenu> column
+    to view an extended table with details about the device setting and performance. Browsing through
     several tabs, you can see lists of <guimenu>Attributes</guimenu>,
-    <guimenu>Metadata</guimenu>, <guimenu>Performance counter</guimenu>, and a
-    graphical <guimenu>Histogram</guimenu> of reads and writes.
+    <guimenu>Metadata</guimenu>, <guimenu>Device health</guimenu>, <guimenu>Performance counter</guimenu>, a
+    graphical <guimenu>Histogram</guimenu> of reads and writes, and
+    <guimenu>Performance details</guimenu>.
    </para>
    <figure>
     <title>OSD Details</title>
     <mediaobject>
      <imageobject role="fo">
-      <imagedata fileref="dash_isd_details.png" width="80%" format="PNG"/>
+      <imagedata fileref="dash_osd_details.png" width="80%" format="PNG"/>
      </imageobject>
      <imageobject role="html">
-      <imagedata fileref="dash_isd_details.png" width="80%" format="PNG"/>
+      <imagedata fileref="dash_osd_details.png" width="80%" format="PNG"/>
      </imageobject>
     </mediaobject>
    </figure>
    <tip>
     <title>Perform Specific Tasks on OSDs</title>
     <para>
-     After you click an OSD node name, its table row changes color slightly,
-     meaning that you can now perform a task on the node. Click the down arrow
-     in the top left of the table heading and select a task to perform, such as
-     <guimenu>Deep Scrub</guimenu>. Optionally enter a required value for the
-     task, and confirm to run it on the node.
-    </para>
+     After you click an OSD node name, the table row is highlighted.
+     This means that you can now perform a task on the node. You can choose to
+     perform any of the following actions: Edit, Create, Scrub, Deep Scrub,
+     Reweight, Mark out, Mark In, Mark Down, Mark Lost, Purge, Destroy, or
+     Delete.
```

Comment:
Should the actions be tagged with [literal] or [guimenu] or something? Perhaps it would look too busy and/or cluttered.